core/src/main/java/io/aiven/klaw/service/BaseOverviewService.java

```diff
@@ -35,6 +35,7 @@ public abstract class BaseOverviewService {
       OBJECT_MAPPER.writerWithDefaultPrettyPrinter();
   public static final TypeReference<ArrayList<TopicHistory>> VALUE_TYPE_REF =
       new TypeReference<>() {};
+  public static final String MASKED_FOR_SECURITY = "Not Authorized to see this.";
```

Comment:
```suggestion
  private static final String MASKED_FOR_SECURITY = "Not Authorized to see this.";
```

can it be private if not used in other classes